hrpsys_choreonoid/launch/add_objects.py

```diff
@@ -46,7 +46,7 @@ def parse_filename(filestr):
         pkgname = ret.group(1)
         #packagepath = commands.getoutput('rospack find %s'%(pkgname))
         packagepath = subprocess.check_output(['rospack', 'find', pkgname])
-        packagepath = packagepath.rstrip('\n')
+        packagepath = packagepath.decode().rstrip('\n')
```

Comment:
confirmed this works on python2 too

Reply:
This script failed to run in melodic. I'll make it compatible with both ros version